### ISSUE TITLE
Startpage.com as Search Engine  Option

### ISSUE THREAD

**Issue by washere** [deepjyoti30/startpage#59]
Startpage as Search Engine  Option:

https://www.startpage.com/

It is different to Duckduckgo as it provides more privacy.
Duckduckgo logs are also cross referencing and kept.

Another major difference is that startpage.com actually uses google. But it does not pass on or log ip numbers.
So basically by using both duckduckgo and startpage we can get the best of both massive databases.

P.S.:
(also in instructions of wiki please add a line: 
"Replace the config.json in source files with your exported config.json and reload from extensions to enable changes."
Took me ten minutes looking for UI/UX options to enable changes made. Will help many. Also can add this line to top of settings + shortcut CTRL+, )

Thanks, as this is a fabulous software.

I think the best overall search engine is startpage.com followed by duckduckgo. And probably for the next ten years or so.
Thanks again.

**Comment by washere** [comment]
To add startpage.com to chrome etc:

* in Settings: search engine section, click [ ADD ] button

* then paste this in query field:

`https://www.startpage.com/sp/search?query=%s`

Then typing in top URL address bar will use Startpage.com anyway too.

**Comment by washere** [comment]
Reason I stopped using duckduckgo in favor of startpage.com recently has been that duckduckgo is getting slower and slower.
Due to commercial packaging in other software/sites & growing user base + increased logging & cross referencing of logs.
It is now the slowest of the top ten engines IMHO.
The only other private contender is startpage.com which unlike duckduckgo logs, is more private too.

**Comment by deepjyoti30** [comment]
@washere Thanks for all the insights. I don't understand what you're trying to ask here. From what I understand, you want `startpage` extension to support `Startpage` search engine, is that correct? If so, we already do support `Startpage` as a search engine. You can just change the `searchEngine` field in `config.json` to the value `Startpage` like following:

```json
{
   "searchEngine": "Startpage"
}
```

> (also in instructions of wiki please add a line:
"Replace the config.json in source files with your exported config.json and reload from extensions to enable changes."
Took me ten minutes looking for UI/UX options to enable changes made. Will help many. Also can add this line to top of settings + shortcut CTRL+, )

Regarding this, I don't understand why you replaced the `config.json` in the source code. You could have just used the import config button that you see in the `settings` view.

To do this, you should open settings by typing `--settings` in the searchbar and pressing enter. On the top left you will see options to `Import` and `Export` config.

**Comment by washere** [comment]
Hi,
Thanks for fast reply.

I opened this because in wiki i saw engine supported values as:

`<DuckDuckGo | Google | Bing |Yahoo>`

As for second issue (related):
I accessed settings by shortcut:

`CTRL+,`

and changes were not updated ! I am a long time coder, tried few times.

I did not access settings with:
 
`--settings`

as you advised, which works perfect in updating as you say.
I do not know why updating with CTRL+, didn't work for me.
Anyway I am closing this & thanks for your great work. Namaste & I'm reminded i really need to visit India again as i love India.

Thanks again buddy.
